### ISSUE TITLE
How to apply to the validation or test set with the model built on training dataset?

### ISSUE THREAD

**Issue by huangmozhilv** [ddsjoberg/dcurves#15]
How to apply to the validation or test set with the model built on training dataset?

**Comment by ddsjoberg** [comment]
Hello! I must have missed this when you posted.

Please review this tutorial for details. https://www.danieldsjoberg.com/dca-tutorial/dca-tutorial-r.html
